src/js/components/Breadcrumb.js

```diff
@@ -1,13 +1,17 @@
 import classNames from "classnames";
 import PropTypes from "prop-types";
 import React from "react";
+import isEqual from "lodash.isequal";
 
 class Breadcrumb extends React.Component {
   shouldComponentUpdate(nextProps) {
     return (
       this.props.isCaret !== nextProps.isCaret ||
       this.props.isIcon !== nextProps.isIcon ||
-      this.props.title !== nextProps.title
+      this.props.title !== nextProps.title ||
+      this.props.instancesCount !== nextProps.instancesCount ||
```

Comment:
These props are specific to `ServiceBreadcrumbs` and we should not be referencing them here.

Reply:
If we don't check them here, we don't trigger a refresh.

Reply:
But then we are messing up something else. We should not need to pass foreign properties to a component for re-rendering although the component does not use the prop in any way. I will look into the issue later / tomorrow.

Reply:
If that's the problem, then it's a problem for `ServiceBreadcrumbs` as well. There we have the service id, so we also don't need them.

Reply:
But if we don't update `Breadcrumb`, then the children also don't get updated.

Reply:
So I played around a little with `shouldComponentUpdate` I unfortunately didn't found a good solution to the problem. Since this is a bug in order to fix it now I propose that we create a follow up issue where we want to refactor this and clean the whole component up and progress the PR as it is?

Reply:
I would rather add a generic optional prop to `Breadcrumbs` and set it from `ServiceBreadcrumbs` based on the values we care about changing. That way we aren't adding `ServiceBreadcrumbs` specific props too the generic `Breadcrumbs` component, and if we run into a similar issue we can use the same solution. I don't have a good idea on the name of said prop though, something generic?

Reply:
Why not just not implement shouldComponentUpdate? It's an optimization, after all.

Reply:
@brandonc , which `shouldComponentUpdate` do you mean? The one in `Breadcrumb` is not added by this PR.

Reply:
Right, I meant getting rid of it. It shouldn't be necessary to update the component

Reply:
As far as I know, we don't mix bug fixing and refactoring.

Reply:
Yep, we don't :) let's fix the bug and follow-up with a refactoring PR

Reply:
Maybe I'm not being understood... This fixes the bug but the feedback has to do with code in the wrong place. Instead of refactoring the fix to satisfy Rodney's concern, you could delete the method altogether and sidestep the issue.

Reply:
@brandonc , you are saying to remove my new code in `Breadcrumb.js` or to remove the entire `shouldComponentUpdate` in `Breadcrumb.js`? If we did the first, it wouldn't work, if we did the second, it would be a refactoring.

Reply:
Removing `shouldComponentUpdate` in `Breadcrumb` would address the root cause of the bug, not be a refactor IMO.

Reply:
Are you sure this wouldn't break something else?

Reply:
If I were a lawyer, I might say "to a reasonable degree of analytical certainty" lol